task_processing/plugins/mesos/retrying_executor.py

```diff
@@ -0,0 +1,60 @@
+import time
+from threading import Thread
+
+from pyrsistent import m
+from six.moves.queue import Queue
+
+from task_processing.interfaces.task_executor import TaskExecutor
+
+
+class RetryingExecutor(TaskExecutor):
```

Comment:
I am also not sure whether we need this. I think we can have either have implicit retries(in the scheduler) or let clients decide if they want to retry.